net-libs/libiscsi/libiscsi-9999.ebuild

```diff
@@ -1,19 +1,16 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit autotools git-r3
 
 DESCRIPTION="iscsi client library and utilities"
 HOMEPAGE="https://github.com/sahlberg/libiscsi"
-SRC_URI=""
 EGIT_REPO_URI="https://github.com/sahlberg/${PN}.git"
```

Comment:
consider putting the 9999 template around this

Reply:
;)